src/main/kotlin/com/github/rushyverse/pvpbox/map/MapImage.kt

```diff
@@ -0,0 +1,75 @@
+package com.github.rushyverse.pvpbox.map
+
+import com.github.rushyverse.pvpbox.PvpboxServer
+import net.minestom.server.coordinate.Point
+import net.minestom.server.coordinate.Pos
+import net.minestom.server.entity.Entity
+import net.minestom.server.entity.EntityType
+import net.minestom.server.entity.metadata.other.ItemFrameMeta
+import net.minestom.server.instance.Instance
+import net.minestom.server.item.ItemStack
+import net.minestom.server.item.Material
+import net.minestom.server.item.metadata.MapMeta
+import net.minestom.server.map.framebuffers.LargeGraphics2DFramebuffer
+import net.minestom.server.network.packet.server.SendablePacket
+import java.awt.geom.AffineTransform
+import java.io.IOException
+import java.io.InputStream
+import javax.imageio.ImageIO
+
+
+internal object MapImage {
+    private var packets: Array<SendablePacket?>? = null
+
+    fun packets(): Array<SendablePacket?>? {
+        return if (packets != null) packets else try {
+            val framebuffer = LargeGraphics2DFramebuffer(5 * 128, 3 * 128)
+            PvpboxServer::class.java.getResourceAsStream("/doge.png")!!.use {
```

Comment:
- No hardcoded value, use the configuration
- According to the size of an image, you should use a buffered reader
```suggestion
            PvpboxServer::class.java.getResourceAsStream("/doge.png")!!.buffered().use {
```
- Take care of the null value ... don't throw an exception

Reply:
Done here https://github.com/Rushyverse/pvpbox/commit/f0084810eb44d1ea5081cf7cf37f85edfeaf1941